Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,272 @@
+Here are the boards of the respective teams: [LaCE Quality](https://github.com/orgs/oppia/projects/3/views/8), [Developer Workflow](https://github.com/orgs/oppia/projects/8/views/11), [Contributor Experience](https://github.com/orgs/oppia/projects/18/views/4).
+
+**Q: What is the total number of contributors that will be accepted?**
+
+A: As many as we think will succeed, though the Google GSoC admins may impose limits based on how they decide to distribute contributor slots among the different open-source organizations.
+
+**Q: I do not have any experience in skill XYZ. Is some certification required?**
+
+A: Try to work on good first issues and take courses online. In the field of software development, it is common to develop experience and expertise as you take up and complete projects successfully. We do not require any formal certification of particular skills.
+
+**Q: Is it okay if I only focus on the frontend or backend?**
+
+A: This probably depends on the project(s) you wish to apply for. However, note that the ability to be effective in both the frontend and backend will open up more opportunities for you, since projects often touch multiple layers of the stack.
+
+**Q: The [Google GSoC FAQ](https://developers.google.com/open-source/gsoc/faq#can_someone_already_participating_in_open_source_be_a_gsoc_contributor) mentions that the program is only for new contributors. I have already contributed to Oppia and I have write access. Can I still participate?**
+
+A: The GSoC program is open to new and beginner contributors to open source and to students. Some of our contributors with write access are still beginner contributors, whereas some of our other contributors with write access will not qualify because they are experienced contributors. If you have only recently received write access, or have been contributing to Oppia for less than a year, you are probably still a beginner contributor. If the previous sentence does not apply to you, and you want to know which group you fall into, please contact **@srijanreddy98**, @U8NWXD or **@aasiffaizal** for a decision.
+
+## Dates and Deadlines
+
+Noteworthy dates for 2023 ([Full Timeline](https://developers.google.com/open-source/gsoc/timeline)):
+
+- **Jan 23 - Feb 7**: Mentoring organizations apply
+- **Feb 22**: Mentoring organizations are announced
+- **Mar 20 - Apr 4**: GSoC contributor application period
+- **May 4**: Accepted GSoC contributors are announced
+- **May 4 - May 28**: Community bonding period
+- **May 29 - Jul 14**: Milestone 1 work period for GSoC contributors
+- **Jul 15 - Jul 16**: External evaluation for M1
+- **Jul 17 - Jul 26**: GSoC contributors can revise M1 according based on th evaluation
```

Comment:
th --> the ?

Also maybe we should write here: "Jul 17 - Jul 26: Optional buffer period for M1 revisions. (Contributors who have passed M1 can begin M2 during this phase.)" Otherwise this seems like a mandatory waiting time.

Reply:
Yeah changed to "Optional buffer period for M1 revisions"

ditto for M2